SynCrtSock.pas

```diff
@@ -6594,6 +6616,17 @@ function THttpServerSocket.GetRequest(withBody: boolean=true): boolean;
         end else
           inc(H);
       end;
+      // remote connection ID
+      L := length(fServer.fRemoteConnIDHeaderUpper);
+      if L<>0 then  begin
+        H := pointer(Headers);
+        for i := 1 to length(Headers) do
+        if IdemPChar(H^,pointer(fServer.fRemoteConnIDHeaderUpper)) and (H^[L]=':') then begin

```

Comment:
why not just include the ':' in the fRemoteConnIDHeaderUpper ?

Reply:
because we need  header without ':' in HTTP.SYS section